modules/enabling-seamless-image-access-and-efficient-use-of-multiple-registry-sources.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-seamless-image-access-and-efficient-use-of-multiple-registry-sources_{context}"]
+= Enabling seamless image access and efficient use of multiple registry sources
+
+By setting up image mirroring for registries, you can ensure faster image pulls that enable seamless image access and efficient use of multiple registry sources in your environment.
```

Comment:
Ensuring faster image pulls may not be accurate or the reason mirroring is setup.

May be best for us not to describe the benefits of mirroring (or even the setup) and leave that to the OCP docs. Instead perhaps we can keep this section focused on specifics of the ACS feature.

Such as:

> ACS supports scanning images from registry mirrors that have been configured via the OCP ImageContentSourcePolicy (ICSP), ImageDigestMirrorSet (IDMS), and/or ImageTagMirrorSet (ITMS) custom resources. (insert link to OCP docs for more info)
>
> Use delegated scanning to automatically scan images from registry mirrors (insert reference to delegated scanning section)